src/app/views/welcome/welcome.vue

```diff
@@ -0,0 +1,49 @@
+import { darken } from 'polished';
+<template>
+	<div class="-welcome-page">
+		<div class="-welcome-col">
+			<section class="-message">
+				<p class="lead">
+					<translate>
+						Before you go game spelunking, let's customize your profile.
+					</translate>
+				</p>
+			</section>
+
+			<section class="-avatar">
+				<app-editable-overlay @click="chooseAvatar()" :disabled="inputDisabled">
+					<translate slot="overlay">Change</translate>
+					<app-user-avatar :user="user" />
+				</app-editable-overlay>
+			</section>
+
+			<section class="-bio">
+				<textarea
+					ref="bio"
+					class="form-control"
+					v-model="user.description_markdown"
+					@change="onBioChanged"
+					:disabled="inputDisabled"
+					:placeholder="$gettext('Tell people about yourself')"
+				></textarea>
```

Comment:
If you want, you can self-close this. vue will know what to do

Reply:
ohey, it does - neat!